### ISSUE TITLE
Adding custom rule with an exception

### ISSUE THREAD

**Issue by jayesh15111988** [realm/SwiftLint#1408]
I am trying to add a custom rule to enforce no space after opening parentheses. Custom rule is defined as follows
```
no_space_after_opening_parentheses:
        name: "No space after opening parentheses"
        message: "Please avoid using space after opening parantheses"
        regex: "(\(\s+)"        
```

However, I want to avoid Swiftlint from warning me when opening brace is followed by newline. Is there any way to avoid warning by doing something like this?

`excludingPattern: "(\(\n)"`

**Comment by marcelofabri** [comment]
This is not possible right now. Maybe you can do that by being clever with only one regex by now.

**Comment by marcelofabri** [comment]
Let's keep it open to track as an enhancement 💯

**Comment by norio-nomura** [comment]
`\s` includes newline.
Can you try `regex: '(\( +)'` instead?

**Comment by jayesh15111988** [comment]
@norio-nomura Thanks, that worked well. Now SwiftLint will catch space after `(` and before `)`, but newlines will be spared in case developer wants to put long arguments on separate lines.

**Comment by norio-nomura** [comment]
Is this already implemented in #1438?

**Comment by norio-nomura** [comment]
It seems not.
